frontend/src/lib/hooks/useKeyboardHotkeys.tsx

```diff
@@ -1,29 +1,64 @@
+import { useValues } from 'kea'
 import { useEventListener } from 'lib/hooks/useEventListener'
 import { DependencyList } from 'react'
-import { Keys } from '~/types'
+import { navigationLogic } from '~/layout/navigation/navigationLogic'
+import { HotKeys, GlobalHotKeys } from '~/types'
 
 export interface HotkeyInterface {
     action: () => void
     disabled?: boolean
 }
 
-export type Hotkeys = Partial<Record<Keys, HotkeyInterface>>
+type LocalHotkeysInterface = Partial<Record<HotKeys, HotkeyInterface>>
+export const useKeyboardHotkeys = (
+    hotkeys: LocalHotkeysInterface,
+    deps?: DependencyList,
+    enableOnGlobal?: boolean
+): void => _useKeyboardHotkeys(hotkeys, deps, enableOnGlobal)
```

Comment:
Because the exported version only allows using hotkeys that are not reserved for global use (i.e. `g` key). This way we make sure that we don't accidentally overwrite a global key in a local scope.

Reply:
Ah, right, the types :).